test/Project.toml

```
@@ -15,6 +16,7 @@ TestItems = "1c621080-faea-4a02-84b6-bbd5e436b8fe"
 Aqua = "0.7, 0.8"
 ExplicitImports = "1.0.1"
 ForwardDiff = "0.10.36"
+JET = "0.9"
```

```suggestion
JET = "0.9.9"
```
To fix Downgrade, we need to bump JET.jl. According to [this comment](https://github.com/aviatesk/JET.jl/issues/665#issuecomment-2395715601), we need [this PR from JET.jl](https://github.com/aviatesk/JET.jl/pull/655), which first appeared in JET.jl v0.9.9.